catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/table/row.view.js

```diff
@@ -63,7 +63,9 @@ module.exports = Marionette.LayoutView.extend({
         this.handleResultThumbnail();
     },
     handleResultThumbnail: function() {
-        if (this.model.get('metacard').get('properties').get('thumbnail')) {
+        var hiddenColumns = user.get('user').get('preferences').get('columnHide');
```

Comment:
An alternative that will always work:
```
if (this.$el.find('> .is-thumbnail').length !== 0 
     && this.model.get('metacard').get('properties').get('thumbnail')) 
```

Reply:
This solution would ensure that if something else hides the column here or elsewhere in the code, this section will still work. I'd argue that Andrew's solution should be preferred.